.markdownlint.json

```
@@ -0,0 +1,6 @@
+{
+  "MD024": {
+    "siblings_only": true
+  },
+  "MD013":false
+}
```

Ooo I didn't know you could do this level of detail?


```suggestion
}

```

---

it works with the vscode plugin, not 100% sure on the gem. I would think they would try and stay compat